wolfcrypt/src/tfm.c

```diff
@@ -1868,9 +1868,23 @@ void fp_montgomery_reduce(fp_int *a, fp_int *m, fp_digit mp)
 
 void fp_read_unsigned_bin(fp_int *a, const unsigned char *b, int c)
 {
+    /* must leave room for extra digit because of fp_mul_2d below */
```

Comment:
Realized I need to remove this -1 comment. I'm going to rebase -> squash merge -> force push.